src/browser/config.ts

```diff
@@ -5,23 +5,25 @@
  * without passing through a transpiler, this file must use ES5.
  */
 
-const fs = require('fs');
-const path = require('path');
-const defaultsDeep = require('lodash.defaultsdeep');
-const sqlectron = require('sqlectron-core');
+import * as fs from 'fs';
```

Comment:
for this (and other imports from node stdlib), given that we have `esModuleInterop ` enabled, we should be able to just do `import fs from 'fs';`? Or is this required to work with babel?

Reply:
I think it was failing even with that enabled, but let me try it once again